security.php

```diff
@@ -177,7 +177,7 @@ function _vip_maybe_temporary_lock_account( $username, $cache_group ) {
 		 *
 		 * @param string $username Username of the request.
 		 */
-		do_action( "{$event_type}_exceeded", $username );
+		do_action( "{$event_type}_limit_exceeded", $username );
```

Comment:
You can have both if you want

Reply:
Nah, I think w are good with just one. The new one (without `_limit` was mistake to be introduced)